### PR TITLE
VFS: Tell the vfs plugin whether we have multiple accounts

### PR DESCRIPTION
**Pull request by TheOneRing**
This allows us to decide on the presentation of the account

### CODE REVIEW

**Comment by TheOneRing** [comment]
Related: https://gitea.owncloud.services/client/client-plugin-vfs-win/pulls/9

File: src/common/vfs.h [hunk 1]
```diff
@@ -62,6 +62,11 @@ struct OCSYNC_EXPORT VfsSetupParams
     QString providerName;
     QString providerVersion;
 
+    /** when registering with the system we might use
+     *  a different presentaton to identify the accounts
+     */
+    bool multipleAccountsKnow = false;
```
Comment (dschmidt) [inline src/common/vfs.h:68]: Strange var name

Comment (TheOneRing) [reply src/common/vfs.h:68]: multipleAccountsKnown... thats we I ask to review my code 🙈

Comment (TheOneRing) [reply src/common/vfs.h:68]: > 
> 
> This needs restarting after adding an account for the display name to update for the old account, right?

Usually only a restart of the client, as the registration is updated every time.

Comment (dschmidt) [reply src/common/vfs.h:68]: Yes, that's what I meant

**Pull request merged**